### DISCUSSION TITLE
Skipped jobs are shown as neutral

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#45894] [Mobile] [answered]
GitHub Mobile beta

Browse to a job run with a skipped job

https://github.com/check-spelling/cassandra/actions/runs/4053939447
![Screenshot_20230131-073508.png](https://user-images.githubusercontent.com/2119212/215763058-4358e0c9-80b9-4c46-bf51-41bbe70a7789.png)

Chrome mobile view 
https://github.com/check-spelling/cassandra/actions/runs/4053939447/jobs/6975160643
![Screenshot_20230131-073558.png](https://user-images.githubusercontent.com/2119212/215763072-a7aedf04-63b6-4fb5-b8f8-f13bd98a8d30.png)

**Comment by alcere** [comment, marked as answer]
Hi @jsoref, I have filled a bug, and the team will evaluate its priority and work on a fix. Thanks for reporting!
